install/tools/ipa-replica-conncheck

```diff
@@ -281,32 +277,86 @@ def configure_krb5_conf(realm, kdc, filename):
 
     krbconf.newConf(filename, opts)
 
+
 class PortResponder(threading.Thread):
 
-    def __init__(self, port, port_type, socket_timeout=1):
+    def __init__(self, ports):
```

Comment:
I would appreciate doc string saying what `ports` is expected to contain. Thanks!